src/lib/pages/backoffice/Series/SeriesDetails/SeriesDetails.js

```diff
@@ -94,7 +93,10 @@ export default class SeriesDetails extends Component {
       content: (
         <Accordion.Content>
           <div id="series-monographs">
-            <SeriesMultipartMonographs />
+            <SeriesMultipartMonographs
+              seriesPid={_get(data, 'pid')}
```

Comment:
we probably have changed the same, in any case `pid` and `mode_of_issuance` are required by the JSON schema, so you don't need the `_get`